common/data/orderUp.json

```diff
@@ -0,0 +1,5 @@
+[
+"1\t12 Nuggets -\t$4.39\n\tSauce: Ketchup\n\tLabel for:Alice",
+"2\tTots -\t$1.99\n\tLabel for:Bob",
+"1\t6 Wings -\t$6.29\n\tWing Seasoning: Sweet Honey BBQ\n\tLabel for:Charlie"
+]
```

Comment:
This has been changed to allow all example orders to be defined in the file, and json would help manipulating that data greatly. We could have the actual order not defined as an array (instead as just a string), but I think it is hard to deal with as is (in terms of developer reading it), and a single string would just exacerbate that. Thoughts?